status-im/status-react
# Update UI for Sign In by Syncing and Add Scan or Enter pairing Code to in app syncing flow

The Onboarding Syncing flow and In app syncing flow is the same except for the background. 
This pr refactors the onboarding syncing screens so that it can be reused for both onboarding and in app syncing flows.

Also I adjust the onboarding syncing page as the background was not showing correctly.

To test:
Ensure the onboarding Syncing flows are working as expect. (i.e "sign in")

Also you can test the syncing flows by going 
profile -> Syncing -> big blue "+" button and then "Scan or enter pairing code" button at the bottom of the "Pair devices to sync" page.

<img width="375" alt="Screenshot 2023-04-11 at 14 30 12" src="https://user-images.githubusercontent.com/22799766/233661020-728f09ff-54ce-4768-a1d1-63fd4c281641.png">
